data/mods/gen4/abilities.js

```diff
@@ -136,25 +135,15 @@ let BattleAbilities = {
 	"intimidate": {
 		inherit: true,
 		onStart(pokemon) {
-			let activated = false;
-			for (const target of pokemon.side.foe.active) {
-				if (target && this.isAdjacent(target, pokemon) &&
-					!(target.volatiles['substitute'] ||
-						target.volatiles['substitutebroken'] && target.volatiles['substitutebroken'].move === 'uturn')) {
-					activated = true;
-					break;
-				}
-			}
+			let activated = pokemon.foes(true).some(foe => !(foe.volatiles['substitute'] || foe.volatiles['substitutebroken'] && foe.volatiles['substitutebroken'].move === 'uturn'));
```

Comment:
```suggestion
			let activated = !pokemon.foes(true).every(foe => !!(
				foe.volatiles['substitute'] || foe.volatiles['substitutebroken'] && foe.volatiles['substitutebroken'].move === 'uturn'
			));
```

Reply:
I'm ok with the indentation but why change this from a single negation to a triple negation?

Reply:
I don't consider `!!` a negation; it's just to shut up TypeScript.

Reply:
You should never make readability worse just to appease TypeScript.

Reply:
I wouldn't "not every" is any more readable than "some not".

Reply:
That's fair. I did this mostly to pull a paren out of the check.

wtb optional chaining already :[